dhall/src/Dhall/TypeCheck.hs

```diff
@@ -2825,7 +2886,7 @@ prettyTypeMessage (RecordTypeMismatch const0 const1 expr0 expr1) =
 
 prettyTypeMessage (FieldCollision k) = ErrorMessages {..}
   where
-    short = "Field collision"
+    short = "Field collision" <> pretty k
```

Comment:
Missing spaces, and I think we want the label to be formatted properly – not sure whether `prettyLabel` would be the right one.

Reply:
Oops. It's probably `prettyLabel`.